test/visual_regression/tests/discover/chart_visualization.js

```diff
@@ -48,10 +49,12 @@ export default function ({ getService, getPageObjects }) {
 
     describe('query', function () {
       this.tags(['skipFirefox']);
+      let renderCounter = 0;
 
       it('should show bars in the correct time zone', async function () {
         await PageObjects.header.awaitGlobalLoadingIndicatorHidden();
         await PageObjects.discover.waitUntilSearchingHasFinished();
+        await find.byCssSelector(`.echChart[data-ech-render-count="${++renderCounter}"]`);
```

Comment:
I like the change, but maybe it is good to wrap it in service since elastic-charts will be used in multiple viz across Kibana. Then we can re-use functions in multiple tests https://github.com/elastic/kibana/commit/8d82ba67bab23f836203b515f03a047a9eb53ff7

Reply:
This suite may look like https://github.com/elastic/kibana/commit/da4f78aa47823371c31dd6ab06b54448695a3ada